plugins/admin_console/kaltura_application_links/admin/scripts/plugin/generic-app-admin-action.phtml

```diff
@@ -0,0 +1,24 @@
+<!-- File: plugins/admin_console/kaltura_application_links/admin/views/scripts/generic-app.phtml -->
+
+<style type="text/css">
+    #sub-header { padding: 5px 20px 0px; }
+    #wrapper { padding: 0 0 0 0; width: 100%; }
+</style>
+<div>
+    <h1> Application generic links </h1>
+    <ul id="applicationLinks"></ul>
+</div>
+<script type="text/javascript">
+    var initialized = false;
```

Comment:
not used

Reply:
probably need 

if (initialized) {
            return;
        }
        initialized = true;
        

in case of double clicks and so